test/lib/mayaUsd/render/mayaToHydra/testTransforms.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2020 Luma Pictures
```

Comment:
Can you please remove the Luma copyright and set 2024 for the copyright from Autodesk ?
Unless you've taken that test from Luma somewhere ?